src/write_log.py

```diff
@@ -0,0 +1,44 @@
+"""
+Write Log File
+
+By: Norma Chang
+"""
+import json
+import time
+import read_log as read
+
+def follower_log(follower):
+    """
+    Logs dictionary data from follower
+    """
+    follower_dict = {
+        'Time' : time.asctime(time.localtime(time.time())), 
```

Comment:
Let's just use epoch time instead of asctime. It'll be way easier to parse in our code.

Reply:
Used epoch time